### ISSUE TITLE
New font really difficult to read

### ISSUE THREAD

**Issue by TheRealBobSmith** [gmertes/NflxMultiSubs#49]
There apparently was an update to the extension released on Feb 24, and now the second language is this thick heavy font that is really difficult to read. It looks like the same font as the primary language. It's not a problem with the primary language because I display the primary language at the largest size with a black background, so it's quite legible. I'm not able to do this with the second language (size nor background), so the second language with the thick font is a white mass that's really difficult to read.

Can you revert this font change? Or add an option to select the font? Thanks.

Edit: This only seems to be happening in Chrome. Font still looks the same in Firefox.

**Comment by gmertes** [comment]
Hi, there was no update recently and this extension doesn't have a text background feature. Are you sure you're using my extension? :)

**Comment by TheRealBobSmith** [comment]
Ohh, you're right. The last update was February 24, 2023. I thought it said February 24, 2024. My bad! :(

The text background on the primary language is a part of Netflix itself, not your extension. You can see how it looks like below.

Maybe the change was in Chrome itself. The secondary language added by your extension is now the same font as the primary language, whereas before it was a thinner, much more legible font. It hasn't changed on Firefox.

Anyway, until very very recently, it used to look like this:
![it used to look like this](https://github.com/gmertes/NflxMultiSubs/assets/25163601/fb2b4e47-514a-4611-bc2a-cac5e8d81f23)

But now it looks like this:
![But now it looks like this](https://github.com/gmertes/NflxMultiSubs/assets/25163601/a4f9b358-7878-41db-8618-3dcc4947ff15)

Agreeing with your comment, it evidently is NOT caused by any change to the extension, so feel free to close the thread. Do let me know if you have any suggestion on getting it to look like the way it was in Chrome again. (I'm unable to use Firefox on the laptop that I use the most for Netflix.)

Thanks and sorry for the bother. Also, thanks for the extension in the first place -- it's super useful, and I wish there were similar extensions for all the streaming services.

**Comment by gmertes** [comment]
Are you sure you don't have any other extensions installed that could be the culprit? I can't reproduce your issue :( 

With latest version of Chrome my subs still look like your first screenshot (both primary and secondary). The extension sets the font style of the secondary subs explicitly, so it shouldn't be influenced by Netflix or Chrome updates. Could you try with Chrome on a different device?

**Comment by TheRealBobSmith** [comment]
I tried it on a different machine, and sure enough, it behaved correctly. I disabled all the extensions on the machine I'm having the issue with, and font of the second language is still massive. I compared the Chrome settings between the two devices and can't find a difference. So bizarre that this problem just randomly started, but I agree that this does not appear to be behavior caused by your extension itself.

BTW, the font is SVG, and I can kind of work around this weird behavior by zooming out of the page when viewing Netflix. This the SVG line not as thick, and makes it more legible.

Thanks for taking a look at this with me, gmertes! Again, really love your extension!
